setup.py

```diff
@@ -32,6 +34,52 @@
 }
 
 
+def check_cmake():
+    try:
+        subprocess.run(["cmake", "--version"], check=True)
+    except:
+        return False
+    return True
```

Comment:
This should capture the output and check for our minimum required cmake version

Reply:
Now also check the cmake version